### DISCUSSION TITLE
Need help to setup Mainnet based on Polygon-Edge Technology

### DISCUSSION THREAD

**Discussion by SYSADMGARG** [0xPolygon/polygon-edge#1156] [Q&A]
Hello Helpers!!!!

Hope you're doing good

Actually Today i'm here to ask one of my query which is regarding polygon-edge based private network.
As per my case i'm running testnet of private network based on polygon-edge technology.Now i'm planning to upgrade this to mainnet from here my problem is starting,i don't have any how to proceed for this no leads where to start.If anyone can help please

Thanks  for listening.

**Comment by laviniat1996** [comment]
Hi @SYSADMGARG ,
A good point to start is our documentation: https://wiki.polygon.technology/docs/edge/overview/
Please let us know if you have any specific questions about it, and I'll gladly reply.

**Reply by SYSADMGARG** [reply]
Hi @laviniat1996 Thanks for replying
Actually i already had a network running on polygon-edge but i wanted to exactly know how can convert or upgrade that network to mainnet anything specific to do as per your knowledge.

Please put some light
Thanks
